.changeset/purple-analysts-depart.md

```diff
@@ -0,0 +1,46 @@
+---
+"@fluidframework/tree": minor
+---
+
```

Comment:
nit: is this missing a section? Or are sections not needed until 2.2?

Reply:
Ah, good catch. Updated.